README.md

```diff
@@ -314,6 +314,50 @@ Creates a query to select an entity with selectorField IN values and keep the OR
 
 Careful, if several entity share the same value, their order is unpredictable.
 
+### transaction helper
+
+Simple helper to manage transaction
+You must retrieve a client with `pool.connect()` to use those.
+
+#### begin
+
+```js
+begin()();
```

Comment:
Where does `Begin` come from ? You should show an import example too. Same for the others